lunch-application/src/theme/Home.vue

```diff
@@ -32,25 +31,26 @@
           </tr>
         </thead>
         <tbody>
-          <tr v-for="row in posts">
+          <tr v-for="row in restaurantData">
```

Comment:
This table is going to show the same restaurants in the same order as the other table. Intentional? (If this is something you're planning to address later, that's totally fine, just slap a TODO comment on there so it's clear there's work still to come)